difflint/utils.py

```diff
@@ -0,0 +1,20 @@
+# TODO License goes here. (MIT)
```

Comment:
I'm not sure what you mean, but I think it's fine to leave this as a TODO. I just wasn't sure until you said that that it was intended as a TODO to be addressed after merging as opposed to before.